DIGNDB.APP.SmitteStop.Jobs/Jobs/EuGateway/DownloadTemporaryExposureKeysEuGatewayJob.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DIGNDB.APP.SmitteStop.Jobs.Config;
+using FederationGatewayApi.Contracts;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace DIGNDB.APP.SmitteStop.Jobs.EuGateway
```

Comment:
This namespace is wrong.
It should correspond to the file location i.e. `DIGNDB.APP.SmitteStop.Jobs.Jobs.EuGateway`